Zigbee2MqttAssistant/appsettings.json

```diff
@@ -6,8 +6,10 @@
   },
   "AllowedHosts": "*",
   "settings": {
-    //"MqttServer": "mqtt",
-    //"MqttUsername": "",
-    //"MqttPassword": ""
+    "MqttServer": "192.168.1.224",
+    "MqttPort": 1883, 
+    "MqttUsername": "",
+    "MqttPassword": "",
+    "Basetopic":  "zigbee2mqtt2" 
```

Comment:
You should remove this from your commit.
Tip: you can locally create a appsettings.Development.json to override those values without modifying the "original" file.

Reply:
My bad.. I really thought I unstaged this before the commit.
I am so new to github. At work we use git and TFS. Fun to learn something new!